README.md

```diff
@@ -5,8 +5,7 @@
 [![Licensed under Apache License version 2.0](https://img.shields.io/github/license/kubevirt/kubevirt.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
 
-**KubeVirt** is a virtual machine management architecture built around
-Kubernetes, implemented as an extension to Kubernetes.
+**KubeVirt** is a virtual machine management add-on for Kubernetes.
```

Comment:
I tend to use lower-case for virtual machine.

For management: I actually considered "management system" (like Doron suggested), but went with "management add-on" because _add-on_ (http://kubernetes.io/docs/admin/addons/) is the term used by Kubernetes to describe ... add-ons/plugins/extensions.

Ideally there will actually be an even nicer mechanism of how to install add-ons to Kubernetes.